takeshape/penny
# Add sample data to pattern

### Test Plan (Steps to test):

1. Import the pattern in this PR as a new project using this URL https://app.takeshape.io/add-to-takeshape?repo=https://github.com/takeshape/penny/tree/sc-10197/chore/add-sample-data-to-pattern/.takeshape/pattern
1. Connect the Shopify service in your new project (can use your own store if you want)
1. Create an api key to use in the next step
1. In a local penny repo (doesn't matter if it's this branch) use your new project's graphql api and api key in your `.env.local`
1. `npm run dev`
1. Make sure the site loads and is generally presentable as an initial state

### Checklist:
